modules/wabt/1.0.37/overlay/include/wabt/config_h.bzl

```diff
@@ -0,0 +1,129 @@
+# Generates `config.h` from `config.h.in`, approximately matching the behavior
+# of CMake.
```

Comment:
Could you add `visibility("private")` to prevent this from being loaded from elsewhere?

Reply:
Done -- used `visibility(["//..."])` to restrict to the `@wabt` module, since this file is now used as a general build utility file.